README.md

```diff
@@ -459,6 +460,14 @@ The current `stacks` predicates support the following `if_this` constructs:
     },
 }
 
+{
+  "if_this": {
+        "scope": "print_event",
+        "contract_identifier": "ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.monkey-sip09",
+        "matches_regex": "(?:^|\\W)vault(?:$|\\W)"
+  }
+}
```

Comment:
An other example that would be pretty useful would be catching both stacking + delegated stacking events on the pox contract.